cloudinit/config/cc_write_files_deferred.py

```diff
@@ -44,6 +49,3 @@ def handle(name, cfg, _cloud, log, _args):
         )
         return
     write_files(name, filtered_files)
-
-
-# vi: ts=4 expandtab
```

Comment:
Yeah good idea throughout. +1 if you wanted a commit to clear that out in general.